content/en/about/faq/traffic-management/statefulsets.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Can Istio be used with StatefulSets and headless Services?
+weight: 30
```

Comment:
Can we update this to 25 or 35, just so we aren't overloading 30?

Reply:
updated